src/category_theory/abelian/exact.lean

```diff
@@ -306,4 +307,26 @@ instance reflects_exact_sequences_of_preserves_zero_morphisms_of_faithful (F : C
 
 end functor
 
+namespace functor
+
+open limits abelian
+
+universes u v
+
+variables {A B : Type u} [category.{v} A] [category.{v} B]
```

Comment:
Sorry for letting this PR  sit for so long. Please wait for #15067 to be merged and then merge master and relax the universe constraints here.

Reply:
Because of `preserves_finite_limits`, both category needs to have the same morphism universe.

Reply:
I think you merged an old master? On the current master, the universe constraints of `preserves_finite_limits` are relaxed.